source/Hardware Guide/Headstages/tether-voltage.rst

```diff
@@ -1,50 +1,46 @@
 .. _tether_voltage:
 
-Tethered Headstage Voltages
+Headstage Voltages
 ==============================
-The voltage on the headstage must be carefully regulated: if its too low the
-headstage will not function reliably. If the voltage is too high sensitive
-components could be damaged. Many issues with the system can be traced back to
-inadequate headstage voltage supply.
+Each ONIX headstage has a required operating voltage that is specified on its
+documentation page. Because ONIX hardware supports headstages that have
+different voltage requirements, it must be changed to to match the requirements
+of the headstage that is plugged into a port. If the headstage voltage is too
+low, it will not function reliably. If the voltage is too high, it
+there will be excess heat dissipation and the headstage may be damaged.
```

Comment:
I would replace:

> If the voltage is too high, it there will be excess heat dissipation and the headstage may be damaged

with something like

>  If the voltage is too high, the headstage will dissipate excess heat and it may be damaged